Textile/Completions/Inline Markup.sublime-completions

```diff
@@ -0,0 +1,116 @@
+{
+    "scope": "text.html.textile",
+    "completions":
+    [
+        {
+            "trigger": "comment",
+            "contents": "<!-- ${1:comment} -->",
+            "annotation": "HTML Comment",
+            "kind": ["snippet", "s", "Snippet"],
+            "details": ""
+        },
+        {
+            "trigger": "notextile",
+            "contents": "<notextile>${1:no-format text}</notextile>",
+            "annotation": "Inline - No formatting",
+            "kind": ["snippet", "s", "Snippet"],
+            "details": ""
+        },
+
+
+
+
+
+        {
+            "trigger": "notextile",
+            "contents": "== ${1:no-format text} ==",
+            "annotation": "Inline - No formatting",
+            "kind": ["snippet", "s", "Snippet"],
+            "details": ""
+        },
```

Comment:
```suggestion
        {
            "trigger": "notextile",
            "contents": "== ${1:no-format text} ==",
            "annotation": "inline",
            "kind": ["snippet", "s", "Snippet"],
            "details": "Span with formatting disabled"
        },
```

This is the style of completion I'd been using. I'm not opposed to what you have, though.